src/main/java/com/nike/vault/client/VaultClient.java

```diff
@@ -287,6 +301,7 @@ protected Response execute(final HttpUrl url, final String method, final Object
         try {
             return gson.fromJson(response.body().string(), responseClass);
         } catch (IOException|JsonSyntaxException e) {
+            logger.error("parseResponseBody: requestUrl={}, response={}", response.request().url(), logResponse(response));
```

Comment:
Then you lose the ability to control what type of log statement it should be, without making a more complex method.